control-plane/subcommand/server-acl-init/rules.go

```diff
@@ -147,12 +147,15 @@ func (c *Command) apiGatewayControllerRules() (string, error) {
 partition "{{ .PartitionName }}" {
   mesh = "write"
   acl = "write"
+  operator = "write"
```

Comment:
Is `operator = "write"` within the partition needed (this feels like more permission than we may want to grant our controller), or is 
```
namespace_prefix "" {
  policy = "write"
}
```
sufficient?

Reply:
+1 to Mike, it's worth investigating if we can avoid operator:write. And I saw this comment: https://github.com/hashicorp/consul-k8s/blob/main/control-plane/subcommand/server-acl-init/rules.go#L40-L41 higher up in the file as well, which indicates operator=write cannot be done within a partition anyways. Not sure if it'll cause issues when deployed (possibly not since you've tested this), but it sounds like if that's the case it should be possible to not include operator:write.

Reply:
After additional testing, I have removed the operator:write permission.

Reply:
Also worth nothing, I'm not sure what was wrong with my env on friday, but on this round of testing, the fix worked with both  api gateway 1.5.1 and 1.5.2

Reply:
I assume you mean API Gateway 0.5.1 and 0.5.1?